include/cutlass/gemm/kernel/intel_pvc_gemm.hpp

```diff
@@ -196,7 +206,7 @@ class GemmUniversal<
   void
   operator()(Params const& params, char* smem_buf) {
```

Comment:
also here  can be like this
```suggestion
  operator()(Params const& params, char* ) {
```

Reply:
We need `smem_buf` in the line below to make sure that we are able to create the epilogue object properly.
`SharedStorage& shared_storage = *reinterpret_cast<SharedStorage*>(smem_buf);`

Reply:
You can probably remove `SharedStorage` entirely from this file.

Reply:
oh, I get it now. `SharedStorage` is needed to create the `FusionCallbacks` in the epilogue